### ISSUE TITLE
Ability to classify devices as network nodes

### ISSUE THREAD

**Issue by ameer1234567890** [jokob-sk/Pi.Alert#452]
Currently only certain types of devices such as switches, routers, APs and modems are classified as network nodes (to which other devices could be connected). However, I have VOIP desk phones which has a LAN input port and a PC port which is the wired to the PC. Thus, this phone acts as a kind of network node, where all PCs are connected to phones and all phones connected to network switches.

So, I request the ability to manually classify devices as network nodes (preferably with a specific number of ports such as that of switches). Could this be implemented as a plugin? If not, perhaps a setting on the device page?

**Comment by jokob-sk** [comment]
Hey! 

Can you test the NETWORK_DEVICE_TYPES setting in the dev build? (backup everything at first)

```
version: "3"
services:
  pialert:
    container_name: pialert
    image: "jokobsk/pi.alert_dev"
    network_mode: "host"
    restart: always
    volumes:
      - ${APP_DATA_LOCATION}/pialert_dev/config:/home/pi/pialert/config
      - ${APP_DATA_LOCATION}/pialert_dev/db:/home/pi/pialert/db
      - ${LOGS_LOCATION}/:/home/pi/pialert/log
    environment:
      - TZ=Europe/Berlin
      - HOST_USER_ID=1036
      - HOST_USER_GID=1000
      - PORT=20211
```

**Comment by ameer1234567890** [comment]
Tested and it works perfectly. Thanks.

**Comment by jokob-sk** [comment]
closing as released in v23.10.2
